gamemode/modules/doorsystem/cl_doors.lua

```diff
@@ -12,7 +12,7 @@ function meta:drawOwnableInfo()
     if doorDrawing == true then return end
 
     local blocked = self:getKeysNonOwnable()
-    local superadmin = ply:IsSuperAdmin()
+    local access = CAMI.PlayerHasAccess(ply, "DarkRP_ChangeDoorSettings") or ply:IsSuperAdmin()
```

Comment:
ply:IsSuperAdmin() or CAMI.PlayerHasAccess(ply, "DarkRP_ChangeDoorSettings")

Reply:
That makes no sense.